### DISCUSSION TITLE
Vagrant Kong Cors Error

### DISCUSSION THREAD

**Discussion by Kibria7533** [Kong/kong#8824] [Help] [answered]
I am running simply a vagrant cluster with kong in my kubernetes.But while making service from admin gui its showing cors error as it is my host machine.Any idea can i run kong with vagrant cluster?

**Comment by jschmid1** [comment, marked as answer]
You may want to try [gojira](https://github.com/Kong/gojira) instead of vagrant (if you're interested in a dev environment for Kong)
